easybuild/easyconfigs/a/ASE/ASE-3.21.1-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'PythonBundle'
+
+name = 'ASE'
+version = '3.21.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://wiki.fysik.dtu.dk/ase'
+description = """ASE is a python package providing an open source Atomic Simulation Environment
+ in the Python scripting language.
+
+From version 3.20.1 we also include the ase-ext package, it contains optional reimplementations
+in C of functions in ASE.  ASE uses it automatically when installed."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', '-Python-%(pyver)s'),
+    ('Flask', '1.1.2', '-Python-%(pyver)s', ('GCCcore', '9.3.0')),
+    ('matplotlib', '3.2.1', '-Python-%(pyver)s'),
+    ('Tkinter', '%(pyver)s', '', ('GCCcore', '9.3.0')),  # needed by GUI of ASE
+    ('spglib-python', '1.16.0', '-Python-%(pyver)s'),  # optional
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('ase', version, {
+        'checksums': ['78b01d88529d5f604e76bc64be102d48f058ca50faad72ac740d717545711c7b'],
+    }),
+    ('ase-ext', '20.9.0', {
+        'checksums': ['a348b0e42cf9fdd11f04b3df002b0bf150002c8df2698ff08d3c8fc7a1223aed'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': ['bin/ase'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+# make sure Tkinter is available, otherwise 'ase gui' will not work
+sanity_check_commands = ["python -c 'import tkinter' "]
```

Comment:
I wonder whether we could add a command for `ASE` itself here? Is there a CLI-part that we can safely run?

Reply:
`ASE` is an interface to many quantum chemistry / quantum physics software packages (see https://wiki.fysik.dtu.dk/ase/). Thus it can be cumbersome to set all the dependencies for verification.